src/test/java/org/jboss/xavier/utils/UtilsTest.java

```diff
@@ -0,0 +1,41 @@
+package org.jboss.xavier.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
```

Comment:
Java6assertions ? this package is **deprecated**

Reply:
changed for `import static org.assertj.core.api.Assertions.assertThat;`